Sources/SwiftLexicalLookup/LookupName.swift

```diff
@@ -121,9 +173,15 @@ import SwiftSyntax
   }
 
   /// Extracts name introduced by `IdentifiableSyntax` node.
-  private static func handle(identifiable: IdentifiableSyntax, accessibleAfter: AbsolutePosition? = nil) -> [LookupName]
-  {
-    if identifiable.identifier.tokenKind != .wildcard {
+  private static func handle(
+    identifiable: IdentifiableSyntax,
+    accessibleAfter: AbsolutePosition? = nil
+  ) -> [LookupName] {
+    if let closureCapture = identifiable as? ClosureCaptureSyntax,
+      closureCapture.identifier.tokenKind == .keyword(.self)
+    {
+      return [.implicit(.self(closureCapture))]  // Handle `self` closure capture.
```

Comment:
You're right. This was a leftover from before when we couldn't create `Identifier` from String, since we can't initialize an `Identifier` with a keyword token. Now, I removed the special case.